localstack/services/dynamodb/dynamodb_listener.py

```diff
@@ -77,7 +77,10 @@ def forward_request(self, method, path, data, headers):
 
             # Fix incorrect values if ReturnValues==ALL_OLD and ReturnConsumedCapacity is
             # empty, see https://github.com/localstack/localstack/issues/2049
-            if data.get('ReturnValues') == 'ALL_OLD' and not data.get('ReturnConsumedCapacity'):
+            # As per amazon docs referenced here
+            # only two values ALL_OLD and None are possible for ReturnValues hence removing the condition.
```

Comment:
Not sure about this comment - I'm seeing this in the docs:
```
Possible values:
    NONE
    ALL_OLD
    UPDATED_OLD
    ALL_NEW
    UPDATED_NEW
```

I think the check `data.get('ReturnValues') == 'ALL_OLD'` was added on purpose - is there a particular reason why we're removing it here?